### ISSUE TITLE
Exception when built into JVM with 2.0.0

### ISSUE THREAD

**Issue by Mohekkus** [ismai117/kottie#25]
Right after adding 2.0.0 dependency while trying to build the project into JVM (without adding any code corresponding to library code) on specifics I gave below, it throw an error.

Additional: **Running on Mac M1 Pro**

[versions]
compose = "1.5.4"
compose-plugin = "1.6.11"
junit = "4.13.2"
kotlin = "2.0.0"

```
Exception in thread "DefaultDispatcher-worker-2" java.lang.UnsatisfiedLinkError: 'long org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallbackKt._nMakeAsFallbackProvider()'
	at org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallbackKt._nMakeAsFallbackProvider(Native Method)
	at org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallbackKt.access$_nMakeAsFallbackProvider(TypefaceFontProviderWithFallback.kt:1)
	at org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallback.<init>(TypefaceFontProviderWithFallback.kt:16)
	at androidx.compose.ui.text.platform.FontCache.<init>(PlatformFont.skiko.kt:201)
	at androidx.compose.ui.text.platform.FontLoader.<init>(PlatformFont.skiko.kt:177)
	at androidx.compose.ui.node.RootNodeOwner$OwnerImpl.<init>(RootNodeOwner.skiko.kt:339)
	at androidx.compose.ui.node.RootNodeOwner.<init>(RootNodeOwner.skiko.kt:147)
	at androidx.compose.ui.node.RootNodeOwner.<init>(RootNodeOwner.skiko.kt)
	at androidx.compose.ui.scene.CanvasLayersComposeSceneImpl.<init>(CanvasLayersComposeScene.skiko.kt:114)
	at androidx.compose.ui.scene.CanvasLayersComposeSceneImpl.<init>(CanvasLayersComposeScene.skiko.kt)
	at androidx.compose.ui.scene.CanvasLayersComposeScene_skikoKt.CanvasLayersComposeScene-3tKcejY(CanvasLayersComposeScene.skiko.kt:93)
	at androidx.compose.ui.scene.CanvasLayersComposeScene_skikoKt.CanvasLayersComposeScene-3tKcejY$default(CanvasLayersComposeScene.skiko.kt:85)
	at androidx.compose.ui.scene.ComposeContainer.createComposeScene(ComposeContainer.desktop.kt:358)
	at androidx.compose.ui.scene.ComposeContainer.access$createComposeScene(ComposeContainer.desktop.kt:84)
	at androidx.compose.ui.scene.ComposeContainer$mediator$3.invoke(ComposeContainer.desktop.kt:142)
	at androidx.compose.ui.scene.ComposeContainer$mediator$3.invoke(ComposeContainer.desktop.kt:142)
	at androidx.compose.ui.scene.ComposeSceneMediator$scene$2.invoke(ComposeSceneMediator.desktop.kt:297)
	at androidx.compose.ui.scene.ComposeSceneMediator$scene$2.invoke(ComposeSceneMediator.desktop.kt:297)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at androidx.compose.ui.scene.ComposeSceneMediator.getScene(ComposeSceneMediator.desktop.kt:297)
	at androidx.compose.ui.scene.ComposeSceneMediator.setCompositionLocalContext(ComposeSceneMediator.desktop.kt:301)
	at androidx.compose.ui.scene.ComposeContainer.setCompositionLocalContext(ComposeContainer.desktop.kt:170)
	at androidx.compose.ui.awt.ComposeWindowPanel.setCompositionLocalContext(ComposeWindowPanel.desktop.kt:76)
	at androidx.compose.ui.awt.ComposeWindow.setCompositionLocalContext(ComposeWindow.desktop.kt:115)
	at androidx.compose.ui.window.Window_desktopKt$Window$13$1.invoke(Window.desktop.kt:649)
	at androidx.compose.ui.window.Window_desktopKt$Window$13$1.invoke(Window.desktop.kt:646)
	at androidx.compose.ui.window.AwtWindow_desktopKt$AwtWindow$2$1.invoke(AwtWindow.desktop.kt:70)
	at androidx.compose.ui.window.AwtWindow_desktopKt$AwtWindow$2$1.invoke(AwtWindow.desktop.kt:69)
	at androidx.compose.runtime.DisposableEffectImpl.onRemembered(Effects.kt:82)
	at androidx.compose.runtime.CompositionImpl$RememberEventDispatcher.dispatchRememberObservers(Composition.kt:1364)
	at androidx.compose.runtime.CompositionImpl.applyChangesInLocked(Composition.kt:992)
	at androidx.compose.runtime.CompositionImpl.applyChanges(Composition.kt:1013)
	at androidx.compose.runtime.Recomposer.composeInitial$runtime(Recomposer.kt:1150)
	at androidx.compose.runtime.CompositionImpl.composeInitial(Composition.kt:649)
	at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:635)
	at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2.invokeSuspend(Application.desktop.kt:221)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)
	Suppressed: kotlinx.coroutines.internal.DiagnosticCoroutineContextException: [StandaloneCoroutine{Cancelling}@3a4f99fc, Dispatchers.Default]
```

**Comment by BBOYGF** [comment]
My Windows 10 system behaves like this too.
The dependencies are like this:
···
  implementation("org.jetbrains.compose.material3:material3-desktop:1.6.11")
    implementation("org.jetbrains.androidx.navigation:navigation-compose:2.7.0-alpha07")
    implementation(compose.desktop.currentOs)
    implementation("io.github.ismai117:kottie:2.0.0")
    runtimeOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core:1.8.1")
    runtimeOnly("org.jetbrains.kotlinx:kotlinx-coroutines-android:1.8.1")

    testImplementation("org.jetbrains.kotlin:kotlin-test")
    testImplementation("org.junit.jupiter:junit-jupiter-api:5.10.0")
    testRuntimeOnly("org.junit.jupiter:junit-jupiter-engine:5.10.0")
···
The error is like this:
···
Exception in thread "main" java.lang.UnsatisfiedLinkError: 'long org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallbackKt._nMakeAsFallbackProvider()'
	at org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallbackKt._nMakeAsFallbackProvider(Native Method)
	at org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallbackKt.access$_nMakeAsFallbackProvider(TypefaceFontProviderWithFallback.kt:1)
	at org.jetbrains.skia.paragraph.TypefaceFontProviderWithFallback.<init>(TypefaceFontProviderWithFallback.kt:16)
	at androidx.compose.ui.text.platform.FontCache.<init>(PlatformFont.skiko.kt:201)
	at androidx.compose.ui.text.platform.FontLoader.<init>(PlatformFont.skiko.kt:177)
	at androidx.compose.ui.node.RootNodeOwner$OwnerImpl.<init>(RootNodeOwner.skiko.kt:339)
	at androidx.compose.ui.node.RootNodeOwner.<init>(RootNodeOwner.skiko.kt:147)
	at androidx.compose.ui.node.RootNodeOwner.<init>(RootNodeOwner.skiko.kt)
	at androidx.compose.ui.scene.CanvasLayersComposeSceneImpl.<init>(CanvasLayersComposeScene.skiko.kt:114)
	at androidx.compose.ui.scene.CanvasLayersComposeSceneImpl.<init>(CanvasLayersComposeScene.skiko.kt)
	at androidx.compose.ui.scene.CanvasLayersComposeScene_skikoKt.CanvasLayersComposeScene-3tKcejY(CanvasLayersComposeScene.skiko.kt:93)
	at androidx.compose.ui.scene.CanvasLayersComposeScene_skikoKt.CanvasLayersComposeScene-3tKcejY$default(CanvasLayersComposeScene.skiko.kt:85)
	at androidx.compose.ui.scene.ComposeContainer.createComposeScene(ComposeContainer.desktop.kt:358)
	at androidx.compose.ui.scene.ComposeContainer.access$createComposeScene(ComposeContainer.desktop.kt:84)
	at androidx.compose.ui.scene.ComposeContainer$mediator$3.invoke(ComposeContainer.desktop.kt:142)
	at androidx.compose.ui.scene.ComposeContainer$mediator$3.invoke(ComposeContainer.desktop.kt:142)
	at androidx.compose.ui.scene.ComposeSceneMediator$scene$2.invoke(ComposeSceneMediator.desktop.kt:297)
	at androidx.compose.ui.scene.ComposeSceneMediator$scene$2.invoke(ComposeSceneMediator.desktop.kt:297)
	at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
	at androidx.compose.ui.scene.ComposeSceneMediator.getScene(ComposeSceneMediator.desktop.kt:297)
	at androidx.compose.ui.scene.ComposeSceneMediator.setCompositionLocalContext(ComposeSceneMediator.desktop.kt:301)
	at androidx.compose.ui.scene.ComposeContainer.setCompositionLocalContext(ComposeContainer.desktop.kt:170)
	at androidx.compose.ui.awt.ComposeWindowPanel.setCompositionLocalContext(ComposeWindowPanel.desktop.kt:76)
	at androidx.compose.ui.awt.ComposeWindow.setCompositionLocalContext(ComposeWindow.desktop.kt:115)
	at androidx.compose.ui.window.Window_desktopKt$Window$13$1.invoke(Window.desktop.kt:649)
	at androidx.compose.ui.window.Window_desktopKt$Window$13$1.invoke(Window.desktop.kt:646)
	at androidx.compose.ui.window.AwtWindow_desktopKt$AwtWindow$2$1.invoke(AwtWindow.desktop.kt:70)
	at androidx.compose.ui.window.AwtWindow_desktopKt$AwtWindow$2$1.invoke(AwtWindow.desktop.kt:69)
	at androidx.compose.runtime.DisposableEffectImpl.onRemembered(Effects.kt:82)
	at androidx.compose.runtime.CompositionImpl$RememberEventDispatcher.dispatchRememberObservers(Composition.kt:1364)
	at androidx.compose.runtime.CompositionImpl.applyChangesInLocked(Composition.kt:992)
	at androidx.compose.runtime.CompositionImpl.applyChanges(Composition.kt:1013)
	at androidx.compose.runtime.Recomposer.composeInitial$runtime(Recomposer.kt:1150)
	at androidx.compose.runtime.CompositionImpl.composeInitial(Composition.kt:649)
	at androidx.compose.runtime.CompositionImpl.setContent(Composition.kt:635)
	at androidx.compose.ui.window.Application_desktopKt$awaitApplication$2$1$2.invokeSuspend(Application.desktop.kt:221)
	at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
	at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:104)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:318)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:773)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:720)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:714)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:399)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:86)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:742)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:203)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:124)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:109)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:90)

···

**Comment by ismai117** [comment]
Sorry for the late response. I've released the 2.0.1 version, try that out!
